src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
This styling is applied to the 'body' tag, which may affect other elements on the page. Consider creating a dedicated class for the stopwatch container and applying these styles to it.